lib/structures/CommandInteraction.js

```diff
@@ -104,7 +108,11 @@ class CommandInteraction extends Interaction {
                 data.member.id = data.member.user.id;
                 this.member = this.channel.guild.members.update(data.member, this.channel.guild);
             } else {
-                const guild = this._client.guilds.get(data.guild_id);
+                let guild = this._client.guilds.get(data.guild_id);
```

Comment:
Is there an auto caching feature I am unaware of? Testing without the changes, there was error on PRIVATE and GUILD (where the guild is a server bot is not in) coming from this line
```js
const guild = this._client.guilds.get(data.guild_id);
this.member = guild.members.update(data.member, guild);
```
![image](https://github.com/user-attachments/assets/69af3e5f-2a59-4e20-bbcc-345276600e8e)
This can be reproduced by reverting my changes and running the example that I added.

Reply:
Right yeah, sorry that slipped my mind. Just create a new instance of Guild in that case